lms/envs/common.py

```diff
@@ -2887,3 +2887,8 @@
 API_ACCESS_FROM_EMAIL = 'api-requests@example.com'
 API_DOCUMENTATION_URL = 'http://edx.readthedocs.org/projects/edx-platform-api/en/latest/overview.html'
 AUTH_DOCUMENTATION_URL = 'http://edx.readthedocs.org/projects/edx-platform-api/en/latest/authentication.html'
+
+# Add Kosovo into django-countries.
+COUNTRIES_OVERRIDE = {
```

Comment:
This is already defined at https://github.com/edx/edx-platform/blob/master/lms/envs/common.py#L2592-L2595.